root/dataframe/dataframe_test.hxx

```diff
@@ -0,0 +1,23 @@
+#include <cstring>
```

Comment:
I acknowledge this header is only going to be included by these tests. I would still place header guards at beggining and end, it's just a good practice (that the original header you link to doesn't follow, I also acknowledge that)